ml-agents/mlagents/trainers/trainer_util.py

```diff
@@ -27,7 +27,27 @@ def __init__(
         param_manager: EnvironmentParameterManager,
         init_path: str = None,
         multi_gpu: bool = False,
+        use_torch: bool = False,
     ):
+        """
+        The TrainerFactory generates the Trainers based on the configuration passed as
+        input.
+        :param trainer_config: A dictionary from behavior name to TrainerSettings
+        :param output_path: The path to the directory where the artifacts generated by
+        the trainer will be saved to.
+        :param train_model: If True, the Trainers will train the model and if False,
+        only perform inference.
+        :param load_model: If True, the Trainer will load neural networks weights from
+        the previous run.
+        :param seed: The seed of the Trainers. Dictates how the neural networks will be
+        initialized.
+        :param param_manager: The EnvironmentParameterManager that will dictate when/if
+        the EnvironmentParameters must change.
+        :param init_path: Path from which to load model.
+        :param multi_gpu: If True, multi-gpu will be used. (currently not available)
+        :param use_torch: If True, the Trainers will all use the Pytorch framework
+        instead of the TensorFlow framewrok.
```

Comment:
```suggestion
        instead of the TensorFlow framework.
```

Reply:
I am shocked that my IDE did not try to stop me. 😲